IRTweaks/IRTweaks/Modules/Misc/DeathChanceStatFix.cs

```diff
@@ -0,0 +1,73 @@
+﻿using BattleTech;
+using Harmony;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Emit;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IRTweaks.Modules.Misc
+{
+    [HarmonyPatch(typeof(Contract), "FinalizeKilledMechWarriors")]
+    static class Contract_FinalizeKilledMechWarriors
+    {
+        public static bool Prepare() => Mod.Config.Fixes.DeathChanceStat;
+
+        public static float CalcChance(float minchance, float basechance, float gutsReduction, float gutsAmount, SimGameState s, Pilot p)
+        {
+            float statmodifier = p.LethalInjuries ? GetStat(s, "LethalDeathChance") : GetStat(s, "IncapacitatedDeathChance");
```

Comment:
LethalDeathChance and IncapactitatedDeathChance aren't company statistics - they're PilotConstants. You want to read these via `s.Constants.Pilot.IncapactitatedDeathChance`.

Reply:
To clarify, in order to read these from company stats, you need to actually instantiate those company stats. "IncapacitatedDeathChance" and "LethalDeathChance" don't exist as statistics by default; they need to be added manually at some point (and should be initialized with their respective SimGameConstant values). SimGameState.InitCompanyStats() is the logical place to do it, but will only take effect on a new game.